### ISSUE TITLE
Can't install addin

### ISSUE THREAD

**Issue by richb-hanover** [ExceLint/ExceLint-addin#45]
**Describe the bug**
I wanted to install the ExceLint-addin so I retrieved the _manifest.xml_ file, saved it to the _wef_ directory as instructed, then opened Excel and used the Add-in to attempt to install it. I saw a spinner for quite a while, then saw this message:

<img width="349" alt="image" src="https://github.com/ExceLint/ExceLint-addin/assets/1094930/580682a4-ce78-4f57-be1d-2972c3227e94">

**To Reproduce**
... See above...

**Expected behavior**
I expected the addin to be installed

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] macOS 12.7.4
 - Browser [e.g. Chrome, Safari] N/A
 - Version [e.g. Excel 2016, Excel Online] Excel for Mac Version 16.84 (24041420)

**Additional context**
@dbarowy @emeryberger I peeked into the _manifest.xml_ file and found this snippet:

```
 <SourceLocation DefaultValue="https://excelint-addin.westus2.cloudapp.azure.com/index.html" />
```

`dig` shows the address for `excelint-addin.westus2.cloudapp.azure.com` to be 255.255.255.255

* Could that plugin be revived on `azure.com`?
* Is there a way to make the plugin available elsewhere? Perhaps on github.com? 

Many thanks.

**Comment by richb-hanover** [comment]
Good news...

1. The ExceLint add-in seems still to work without incident with Excel for Mac (current version)

2. I was able to build the add-in locally (I needed to use node 16.4.0 to avoid conflicts/outdated packages)

3. I also created a Dockerfile that builds a container for testing on localhost. This helps people avoid the need to install all kinds of software on their computer to make it work. Would you be interested in a PR?

Finally, any thoughts about making the add-in available from a different server? 

Thanks again for this wonderful tool.

Rich Brown
Lyme, NH

**Comment by richb-hanover** [comment]
Solved with #48. ExceLint add-in is now served from a Github Page from this repo
